ckan/docker-entrypoint.d/10-setup-harvest.sh

```diff
@@ -1,9 +1,20 @@
+set -e
+
 echo "Init Harvest database tables"
 ckan harvester initdb
 
 echo "turn on gather and fetch workers"
 ckan harvester fetch-consumer &
+ckan harvester fetch-consumer &
 ckan harvester gather-consumer &
 
-echo "check harvest job completion every minute, see harvest-check-cron"
-crond -b -l 8 &
+echo "check harvest job completion every 10 secs"
+check_harvester () {
+  while true
+  do
+    ckan harvester run &> /tmp/harvester_run.log
+    sleep 10
+  done
+}
+
+check_harvester &
```

Comment:
Are you saying that running every 10 secs is too often?  When we run this manually, we can always change the sleep.

Reply:
I'm saying it's too often if we want to pull down a lot of external data, but for 95% of use cases it will be fine. With thousands of datasets, that will cause a lot of load. In case anyone pulls in larger sources to look into user bugs, etc to be aware. But I think it's fine for now to speed up the tests.